samples/sampler/stories/default/Dropdown.js

```diff
@@ -33,3 +33,52 @@ storiesOf('Agate', module)
 			text: 'The basic Dropdown'
 		}
 	);
+
+storiesOf('Agate QA.Dropdown', module)
+	.add(
+		'with 4 directions',
+		() => {
+			const itemCount = number('items', Config, {range: true, min: 0, max: 50}, 5);
+			const items = (new Array(itemCount)).fill().map((i, index) => `Option ${index + 1}`);
+
+			return (
+				<>
+					<Dropdown
+						direction="left"
+						onSelect={action('onSelect')}
+						style={{position: 'absolute', top: 0, right: 0, width:'25%'}}
+						title="Left dropdown"
+					>
+						{items}
+					</Dropdown>
+					<Dropdown
+						direction="right"
+						onSelect={action('onSelect')}
+						style={{position: 'absolute', top: 0, left: 0, width:'25%'}}
+						title="Right dropdown"
+					>
+						{items}
+					</Dropdown>
+					<Dropdown
+						direction="down"
+						onSelect={action('onSelect')}
+						style={{position: 'absolute', top: 0, left: '50%', transform: 'translateX(-50%)', width:'25%'}}
+						title="Down dropdown"
+					>
+						{items}
+					</Dropdown>
+					<Dropdown
+						direction="up"
+						onSelect={action('onSelect')}
+						style={{position: 'absolute', top: ri.scaleToRem(350), width:'25%'}}
```

Comment:
Could we change `350` to `440` to show the dropdown properly in a viewport because it height is 435px?

Reply:
![image](https://user-images.githubusercontent.com/4239873/87381691-42a1f080-c5d0-11ea-853f-c2c6661bb093.png)


I found there was one issue.
After expanding "Up dropdown", if clicking the "Right dropdown" to be expanded,
the "Right dropdown" expands under the "Up dropdown". So the menus in the "Right dropdown" can't be seen.

Reply:
I created the FORTECH-83 issue for the issue above. Let's solve it separately.